### PR TITLE
Support deploying a few aurora engines for silos testing

### PR DESCRIPTION
**Pull request by olga24912**

### CODE REVIEW

**Review by birchmd — APPROVED** [review]

File: aurora-rust-sdk/aurora-sdk-integration-tests/src/aurora_engine/mod.rs [hunk 1]
```diff
@@ -26,13 +26,13 @@ pub struct AuroraEngine {
     pub inner: Contract,
 }
 
-pub async fn deploy_latest(worker: &Worker<Sandbox>) -> anyhow::Result<AuroraEngine> {
+pub async fn deploy_latest_silo(worker: &Worker<Sandbox>, account_id: &str) -> anyhow::Result<AuroraEngine> {
     let wasm = repo::AuroraEngineRepo::download_and_compile_latest().await?;
```
[the remark this answers is not in the document]
Comment (karim-en) [reply aurora-rust-sdk/aurora-sdk-integration-tests/src/aurora_engine/mod.rs:30]: @aleksuss What do you propose? Adding a method new method `deploy_silo` and passing the wasm to it?

**Review by karim-en — APPROVED** [review]

File: aurora-rust-sdk/aurora-sdk-integration-tests/src/aurora_engine/mod.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (aleksuss) [reply aurora-rust-sdk/aurora-sdk-integration-tests/src/aurora_engine/mod.rs:30]: E.g. there could be added param to the `download_and_compile_latest`.
```rust
enum AuroraContract {
    Main,
    Silo,
}
```
I use the prefix `silo-` for silo's tags.
`AURORA_LATEST_SILO_VERSION = format!("silo-{AURORA_LATEST_VERSION}");`

[the remark this answers is not in the document]
Comment (aleksuss) [reply aurora-rust-sdk/aurora-sdk-integration-tests/src/aurora_engine/mod.rs:30]: Also, we have built binaries which could be downloaded from github. 
```bash
AURORA_LATEST="2.10.2"
AURORA_SILO_LATEST="silo-$AURORA_LATEST"
https://github.com/aurora-is-near/aurora-engine/releases/download/$AURORA_LATEST/aurora-mainnet.wasm
https://github.com/aurora-is-near/aurora-engine/releases/download/$AURORA_SILO_LATEST/aurora-mainnet.wasm
```

[the remark this answers is not in the document]
Comment (aleksuss) [reply aurora-rust-sdk/aurora-sdk-integration-tests/src/aurora_engine/mod.rs:30]: Ok. No problem.

**Comment by aleksuss** [comment]
By the way, it can't be merged because the commit is unsigned
cc @olga24912

**Review by sept-en — APPROVED** [review]

**Review by aleksuss — APPROVED** [review]

**Pull request merged**